requirements/requirements-linux.txt

```diff
@@ -1,34 +1,34 @@
-astroid==2.2.3
+astroid==2.2.2
 beautifulsoup4==4.7.1
 bs4==0.0.1
 certifi==2018.11.29
 chardet==3.0.4
 Click==7.0
 gTTS==2.0.3
 gTTS-token==1.1.3
-idna==2.8
-isort==4.3.10
+idna==2.5
+isort==4.3.9
 lazy-object-proxy==1.3.1
-lxml==4.3.2
+lxml==4.3.1
 mccabe==0.6.1
+newsapi==0.1.1
+newsapi-python==0.2.3
+pgi==0.0.11.2
 pkg-resources==0.0.0
 playsound==1.2.2
-<<<<<<< HEAD
 PyAudio==0.2.11
 pylint==2.3.1
-=======
-pycairo==1.18.0
->>>>>>> 8b596ead06505c11e95e62beab295d7c9e8ab8bc
 PyouPlay==0.1.3
 PyQt5==5.12
 PyQt5-sip==4.19.14
-requests==2.21.0
+requests==2.17.1
 ruamel.yaml==0.15.89
 six==1.12.0
 soupsieve==1.8
 SpeechRecognition==3.8.1
 typed-ast==1.2.0
-urllib3==1.24.1
+urllib3==1.21.1
```

Comment:
why 24 --> 21 ?

Reply:
I don't remember it exactly but one of the new packages works only with the 1.21.1 version.